.gen/go/cadence/tchan-cadence.go

```diff
@@ -135,6 +135,8 @@ func (c *tchanWorkflowServiceClient) GetWorkflowExecutionHistory(ctx thrift.Cont
 			err = resp.InternalServiceError
 		case resp.EntityNotExistError != nil:
 			err = resp.EntityNotExistError
+		case resp.ServiceBusyError != nil:
```

Comment:
Minor: Will this error be used for other kinds of scenarios too or is it only throttling? I have often seen a specific "RequestThrottledError" which makes it clear to the client why this is happening.